source/Game.cpp

```diff
@@ -0,0 +1,199 @@
+#include"Game.hpp"
+
+#include<sstream>
+#include<iostream>
+#include<cctype>
+#include<string>
+#include"Error.hpp"
+
+namespace dungeon {
+
+Game::Game(): isEnd(false) {
+  initialize();
+}
+
+void Game::welcome() {
+  ios.clear();
+  ios.print("hello, world");
+  ios.pause();
+}
+
+void Game::selectClass() {
+  ios.clear();
+  ios.print("Select your Class");
+  ios.print("A. Saber");
```

Comment:
Tight coupling code

Reply:
I'm doing that, changing into using ostream directly.

Reply:
Tight coupling code means you hard-coded some child classes in the code, which makes your code non-flexible and tightly depends on those variant classes.